indico/modules/rb_new/client/js/modules/blockings/BlockingModal.jsx

```diff
@@ -324,6 +391,15 @@ class BlockingModal extends React.Component {
                                        isEqual={_.isEqual}
                                        render={this.renderRoomSearchField}
                                        disabled={mode === 'view' || submitting || submitSucceeded} />
+                                {blocking.canEdit && blocking.blockedRooms.length !== 0 && (
```

Comment:
`blocking.canEdit` isn't correct here; you do not need to have edit access on the blocking to approve/reject the blocking for your rooms